### ISSUE TITLE
Able to use MongoDB 3.4.x?

### ISSUE THREAD

**Issue by dvanwinkle** [ParsePlatform/parse-server#3517]
Are we able to start using the latest MongoDB seeing as parse.com is now closed? If I remember right, the only reason why the 3.0.x requirement was there in the first place was due to Parse's servers only supporting that version.

@flovilmart @acinader

**Comment by steven-supersolid** [comment]
The tests currently run against mongo 3.0.8 and 3.2.6. We could update to run against 3.4.x, possibly replacing the 3.2.6 tests if there are no problems. I don't foresee anything going wrong but we are stuck on an older version of the mongodb driver and it is not trivial to update.

If you want to do a quick check you can modify package.json to use 3.4.x and run parse-server tests locally. I recommend you do this also for your own code in case a problem is not covered by a parse-server test.

**Comment by flovilmart** [comment]
> we are stuck on an older version of the mongodb driver 

Not anymore ;)

**Comment by TylerBrock** [comment]
I can take a look at some point.

**Comment by nowfalsalahudeen** [comment]
am using 3.4.2

**Comment by markmssd** [comment]
I am using Mongo v3.4 too, on Mongo Atlas, and works without any problem. 👍

**Comment by flovilmart** [comment]
So we can close that!

**Comment by madsb** [comment]
@markmssd I'm testing out the same setup as you (Atlas/v3.4), but I'm running into issues. I think it's related to Atlas not supporting sparse indexes, which results in errors when comparing unique values such as email. Have you experienced the same, or did you find a solution with partial indexes?

**Comment by lockwobr** [comment]
This seems old but the [docs](http://docs.parseplatform.org/parse-server/guide/#database) does not say that 3.4 works? Do the docs just need updating, or am I reading this wrong?

**Comment by flovilmart** [comment]
even 3.6 works :)

**Comment by marcelocotrim** [comment]
3.6 is working perfectly!

**Comment by fmendoza** [comment]
What about 4?

**Comment by karthikSmartserv** [comment]
> 3.6 is working perfectly!

No it's not working at all, it sucks -_-

**Comment by davimacedo** [comment]
Should be working in 4 as well: https://travis-ci.org/parse-community/parse-server/jobs/561149679

**Comment by LilMoke** [comment]
Hello, I stumbled across this post in search of some help with getting ParseServer to work with MongoDB.  Heroku is ending the mLab add-on so I am trying to connect directly to MongoDB.

I have set up a database and pointed my database_uri to the url to the DB.   can access the DB via Robo3T so I think the DB is ok.  Parse seems to start with no errors in VERBOSE, but when I try to navigate to the parse dashboard I get the message: {"error":"unauthorized"} .

The Database URI is slightly different with 3.6 but other than that everything I have came across indicates it should work.

Is the mongodb+srv:// an acceptable format for the uri?
Is there other reasons for this error?
